source/_docs/dns.md

```diff
@@ -11,26 +11,18 @@ contributors: alexfornuto
 
 ![DNS Reccomendations](/source/docs/assets/images/dashboard/domain-added.png)
 
-While Pantheon does not offer DNS management services, we can help you to understand how DNS works, and configure your domain to point to your Pantheon site. We also offer DNS Provider-specific instructions for several common DNS managers.
+While Pantheon does not offer DNS management services, we can help you to understand how DNS works, and configure your domain to point to your Pantheon site.
 
-<div class="panel panel-drop panel-guide" id="accordion1">
-  <div class="panel-heading panel-drop-heading">
-    <a class="accordion-toggle panel-drop-title collapsed" data-toggle="collapse" data-parent="#accordion" data-proofer-ignore data-target="#unique-anchor">
-      <h3 class="info panel-title panel-drop-title" style="cursor:pointer;"><span style="line-height:.9" class="glyphicons glyphicons-wrench"></span>DNS Provider-Specific Instruction</h3>
-    </a>
-  </div>
-<div id="unique-anchor" class="collapse" markdown="1" style="padding:10px;">
-### DNS Provider-Specific Instruction {.info}
+## DNS Provider Instructions
+
+We offer DNS provider-specific instructions for several common DNS managers:
```

Comment:
does it make sense to standardize on what we call this? In the product and launch guide I think we use "DNS host"

Reply:
Updated.